xstate-chatbot/nodejs/src/machine/chat-machine.js

```diff
@@ -101,6 +102,10 @@ const chatStateMachine = Machine({
               cond: (context) => context.intention == 'rmo',
               target: '#rmoMobileNumber',
             },
+            {
+              cond: (context) => context.intention == 'bedsavailability',
+              target: '#nodalOfficer',
```

Comment:
target to hospitalFlow. As initial of hospitalFlow is nodalOfficer it will go to the nodalOfficer. 
The way it is right now, it will skip following statements
context.slots.hospital = {};    
context.slots.previoushospitaldata = {};

Reply:
changed as per your suggestions